indico/modules/events/papers/models/files.py

```diff
@@ -49,11 +50,11 @@ class PaperFile(StoredFileMixin, db.Model):
         nullable=True
     )
 
-    contribution = db.relationship(
+    _contribution = db.relationship(
```

Comment:
No strong opinion here. On one hand, It might be considered cleaner to access the parent (`Contribution`) only from it's "virtual" direct children (`Paper`), on the other hand it doesn't harm since it's technically an ancestor of both (`Paper` and `PaperFile`).